apps/fluent-tester/src/FluentTester/TestComponents/ButtonExperimental/ButtonVariantTestSection.tsx

```diff
@@ -1,20 +1,58 @@
 import { Button, CompoundButton } from '@fluentui-react-native/experimental-button';
 import * as React from 'react';
-import { View } from 'react-native';
+import { View, Platform, processColor } from 'react-native';
 import { commonTestStyles, stackStyle } from '../Common/styles';
 
 export const ButtonVariantTest: React.FunctionComponent<{}> = () => {
-  return (
-    <View style={[stackStyle, commonTestStyles.view]}>
-      <Button content="Default" style={commonTestStyles.vmargin} />
-      <Button primary content="Primary" style={commonTestStyles.vmargin} />
-      <Button ghost content="Ghost" style={commonTestStyles.vmargin} />
-      <Button fluid content="Fluid" style={commonTestStyles.vmargin} />
-      <Button primary fluid content="Fluid Primary" style={commonTestStyles.vmargin} />
-      <Button ghost fluid content="Fluid Ghost" style={commonTestStyles.vmargin} />
-      {/* <CompoundButton content="Default" secondaryContent="Compound" style={commonTestStyles.vmargin} />
-      <CompoundButton primary content="Primary" secondaryContent="Compound" style={commonTestStyles.vmargin} /> */}
-      <CompoundButton ghost content="Ghost" secondaryContent="Compound" style={commonTestStyles.vmargin} />
-    </View>
-  );
+  if (Platform.OS == 'macos') {
+    const CustomizedNativeButton = Button.customize({
+      contentTintColor: processColor('white'),
+      restBackgroundColor: processColor('orange'),
```

Comment:
You shouldn't need to call `processColor`. That is an internal React-Native API and when I used it in Shimmer, it was a bug. If you set `contentTintColor` and `restBackgroundColor` as `string` types instead of `Number`, RCTConvert should convert them to proper NSColors. I can follow up offline if that causes problems

Reply:
That didn't work for mac. Could it be RCTConvert doesn't support NSColor?